src/main/java/com/epam/ta/reportportal/binary/impl/AttachmentBinaryDataServiceImpl.java

```diff
@@ -128,9 +128,10 @@ public Optional<BinaryDataMetaInfo> saveAttachment(AttachmentMetaInfo metaInfo,
     } catch (IOException e) {
       LOGGER.error("Unable to save binary data", e);
     } finally {
-      if (file instanceof CommonsMultipartFile) {
+      // TODO: uncomment and fix later !!!
```

Comment:
What needs to be fixed?

Reply:
Thanks for pointing on this. I can asume that the previous purpose of that was to handle temporary files created while preparing demodata, specificaly attachments for the logs. And since Spring libraries migrated to their own implementaion of file handling instead of apache libraries - this code is not releavant anymore and can be safely removed.